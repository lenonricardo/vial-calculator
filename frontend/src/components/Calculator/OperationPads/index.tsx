import {
  Wrapper,
  PrimaryPad,
  SecondaryPad,
  PrimaryGradientPad,
  SecondaryGradientPad
} from './styles'

const OperationPads = () => (
  <>
    <Wrapper>
      <SecondaryPad>%</SecondaryPad>
      <SecondaryPad>(</SecondaryPad>
      <SecondaryPad>)</SecondaryPad>
      <SecondaryPad>√</SecondaryPad>
      <SecondaryPad>xʸ</SecondaryPad>
    </Wrapper>

    <Wrapper>
      <PrimaryPad>÷</PrimaryPad>
      <PrimaryGradientPad>x</PrimaryGradientPad>
      <SecondaryGradientPad>-</SecondaryGradientPad>
      <SecondaryGradientPad>+</SecondaryGradientPad>
      <SecondaryGradientPad>=</SecondaryGradientPad>
    </Wrapper>
  </>
)

export default OperationPads
