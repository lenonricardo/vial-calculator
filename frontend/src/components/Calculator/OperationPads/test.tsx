import { render } from '@testing-library/react'

import OperationPads from '.'

describe('<OperationPads />', () => {
  it('should render the component', () => {
    const { container } = render(<OperationPads />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
