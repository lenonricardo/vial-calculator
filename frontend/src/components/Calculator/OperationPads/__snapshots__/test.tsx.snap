// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OperationPads /> should render the component 1`] = `
.c1 {
  width: 90px;
  height: 90px;
  color: var(--font-pads);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 35px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c2 {
  background-color: var(--secondary-operations);
}

<div
  class="c0"
>
  <div
    class="c1 c2"
  >
    %
  </div>
  <div
    class="c1 c2"
  >
    (
  </div>
  <div
    class="c1 c2"
  >
    )
  </div>
  <div
    class="c1 c2"
  >
    √
  </div>
  <div
    class="c1 c2"
  >
    xʸ
  </div>
</div>
`;
